sdk/state/close.go

```diff
@@ -91,9 +91,8 @@ func (c *Channel) ConfirmCoordinatedClose(cc CoordinatedClose) (coordinatedClose
 	}
 	fullySigned = true
 
-	// TODO - merge instead of overwrite, similar to ConfirmProposal
-	c.coordinatedClose = cc
-	return cc, fullySigned, nil
+	c.coordinatedClose.CloseSignatures = appendNewSignatures(c.coordinatedClose.CloseSignatures, cc.CloseSignatures)
```

Comment:
We never seem to set `c.coordinatedClose` in the participant who doesn't call propose. We only appear to modify it. I guess it doesn't have any other fields, but I think we should still do the following, and I expect in the future we might need to store other things in it?

```suggestion
	c.coordinatedClose = CoordinatedClose{CloseSignatures: appendNewSignatures(c.coordinatedClose.CloseSignatures, cc.CloseSignatures)}
```

If you feel strongly against this feel free to move ahead without it.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/88/commits/899c9a46671dc43ca318575f74818f2d0d351a7c

probably this will be changed again in the PR I'm making for not using CoordinatedClose. But atm I agree this is more correct